bigtable/client/filters.h

```diff
@@ -410,34 +409,93 @@ class Filter {
    *
    * These filters compose several filters to build complex filter expressions.
    */
-  /// Return a filter that selects between two other filters based on a
-  /// predicate.
-  // TODO(#30) - implement this one.
+  /**
```

Comment:
Should these comments be `///` or `/** ... */`? Has this been decided?

Reply:
`///` for one liners, `/** ... */` otherwise.  The style guide PR (#98) includes that [link](https://github.com/GoogleCloudPlatform/google-cloud-cpp/pull/98/files#diff-3ef928dd07fd874e279d2145cd6d6831R660)